### DISCUSSION TITLE
Potential New Way to Auto Start PersistentWindows?

### DISCUSSION THREAD

**Discussion by Thisisaaccountyes** [kangyu-california/PersistentWindows#219] [Ideas]
New to GitHub, and I'm not sure how to contact the creator themselves or where else to put this. 
So here we go!

I had an issue I recently figured out how to fix #210 
I left a extra message/fix for it but am not sure how to ask the creator them self if this potential method was ever used and or thought of. If anyone has any information about it, please send it here! I'd love to understand the potentials pros or cons of doing this method.

I guess for both how to conduct myself on GitHub for future reference and the solution!

**Comment by Thisisaaccountyes** [comment]
Note, if this post sounds weird, sorry! I Spent like 2 hours fixing this and then trying to figure out how to contact the creator. Finally coming here.

**Comment by kangyu-california** [comment]
The method of creating a shortcut in startup folder was initially suggested, however the method won't work if user want to run PW in admin privilege.
That's why I recommend task scheduler method for power users.

**Reply by Thisisaaccountyes** [reply]
I see, does make sense, the startup version I am using basically does the double click for you instead of actually running some kind of script. If you have it on admin, it doesn't work cause it doesn't accept the startup prompt for you. But I need to ask, why run admin? I never needed to on my first pc. Never ran into that problem once. Then again, I run into a slew of different problems. One pc to another.

Perhaps one day it might even be put on the known issues section! Here's to hoping I saved a lot of people time and misery.
